src/Microsoft.Azure.SignalR.Common/Auth/AzureActiveDirectoryOptions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Azure.SignalR
+{
+    public class AzureActiveDirectoryOptions
+    {
+        public readonly X509Certificate2 ClientCert;
```

Comment:
`public` means it is exposed to user, is it expected? or `internal`? and same questions apply to below properties.

Reply:
I think there is no reason that we don't allow our users to read a property set by themselves?